gui/src/components/LinkButton.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import StyledButton from '@material-ui/core/Button'
+
+export default props => (
```

Comment:
I feel like this should allow classes to be assigned for styling. How do you feel about making the `SIGN IN AGAIN` link white underline title case?